cram_common/cram_manipulation_interfaces/src/manipulation-interfaces.lisp

```diff
@@ -65,3 +65,9 @@ make CRAM_DESIGNATORS package depend on CRAM_MANIPULATION_INTERFACES,
 and man-int is way too high level to make it a dependency of CRAM_CORE.
 This hijacking is kind of an ugly hack that Gaya feels bad about :(."
   (get-location-poses desig))
+
+(defgeneric get-container-opening-distance (container-designator)
```

Comment:
The parameter should be ``container-type'' not a whole designator, such that we can use ``eql'' method overloading stuff.